DataRepo/models.py

```diff
@@ -319,6 +323,85 @@ class PeakGroup(models.Model):
     def atom_count(self, atom):
         return atom_count_in_formula(self.formula, atom)
 
+    @cached_property
+    def total_abundance(self):
+        """
+        Total ion counts for this compound. Accucor provides this in the tab
+        "pool size". Calculated by summing the corrected_abundance of all
+        Measurements for this compound.
+        """
+        return self.peak_data.all().aggregate(
+            corrected_abundance=Sum("corrected_abundance")
+        )["corrected_abundance"]
```

Comment:
or is that also equivalent to?
```suggestion
        return self.peak_data.all().aggregate(Sum("corrected_abundance"))["corrected_abundance__sum"]
```

Reply:
You got it (and that is arguably cleaner, though I didn't like the implicit dictionary key). For reference see the [Django aggregation docs](https://docs.djangoproject.com/en/3.2/topics/db/aggregation/).

Also, you can try things out like this:
```python
In [5]: from DataRepo.models import *                                                                                                                                                                      

In [6]: pg = PeakGroup.objects.get(id=1)

In [7]: from django.db.models import Sum

In [8]: pg.peak_data.all().aggregate(Sum("corrected_abundance"))
Out[8]: {'corrected_abundance__sum': 9599112.684141196}

In [9]: pg.peak_data.all().aggregate(Sum("corrected_abundance"))["corrected_abundance__sum"]
Out[9]: 9599112.684141196

In [10]: pg.peak_data.all().aggregate(ca=Sum("corrected_abundance"))
Out[10]: {'ca': 9599112.684141196}

In [11]: pg.peak_data.all().aggregate(ca=Sum("corrected_abundance"))["ca"]
Out[11]: 9599112.684141196
```